### PR TITLE
stop the green/black fade

### PR DESCRIPTION
**Pull request by astronaut-wannabe**
demo: https://crimethinc-staging-pr-685.herokuapp.com

### CODE REVIEW

**Comment by TheConObs** [comment]
Damn. I thought I had taken care of that with the change to just black for the fader color list. I bet I missed the default background-color. Sorry gents.

File: app/assets/stylesheets/components/_header.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .site-header {
   align-items: center;
-  background-color: #76c261;
```
Comment (TheConObs) [inline app/assets/stylesheets/components/_header.scss]: Ah yep.

Comment (astronaut-wannabe) [reply app/assets/stylesheets/components/_header.scss]: no worries, figured it was just a missed debugging thing (like `border: solid 1px red;`)

Comment (TheConObs) [reply app/assets/stylesheets/components/_header.scss]: Ugh yeah I hate when that happens ;-)

**Pull request merged**